packages/inference/src/HfInference.ts

```diff
@@ -720,6 +730,20 @@ export class HfInference {
 		return res;
 	}
 
+	/**
+	 * Answers a question on a document image. Recommended model: impira/layoutlm-document-qa.
+	 */
+	public async documentQuestionAnswer(args: ImageQuestionAnswerArgs, options?: Options): Promise<QuestionAnswerReturn> {
+		return (await this.request<[QuestionAnswerReturn]>(args, options))?.[0];
```

Comment:
hi @vvmnnnkv could we also add a validation here for the single response value here and for visualQuestionAnswer? thanks

```js
const res = await this.request<[QuestionAnswerReturn]>(args, options))?.[0];
const isValidOutput = Array.isArray(res) && res.every((x) => typeof x.answer === "string" && typeof x.score === "number")
if (!isValidOutput) {
	throw new TypeError("Invalid inference output: output must be of type Array<answer: string, score: number>");
}
```